src/main/webapp/app/exercises/shared/plagiarism/plagiarism-split-view/split-pane-header/split-pane-header.component.scss

```diff
@@ -50,3 +50,7 @@
         overflow: scroll;
     }
 }
+
+.split-pane-header-file.hover {
+    background-color: lightskyblue;
```

Comment:
I am afraid, this is not one of our default colors. Could you use one of the defined colors?